lib/assets/javascripts/twine.coffee

```diff
@@ -265,6 +282,9 @@
   isKeypath = (value) ->
     value not in ['true', 'false', 'null', 'undefined'] && keypathRegex.test(value)
 
+  isDataAttribute = (value) ->
+    value[0] == 'd' && value[1] == 'a' && value[2] == 't' && value[3] == 'a' && value[4] == '-'
```

Comment:
nitpick: stylistically, easier to read vertically if formatted like
```coffee
value[0] == 'd' &&
value[1] == 'a' &&
...
```

❤️ that this also short circuits fast too